src/AaveV2DelegatesGasRebate_20230703/AaveV2EthBUSDIR_20230602Test.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import 'forge-std/Test.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {ProtocolV2TestBase, ReserveConfig} from 'aave-helpers/ProtocolV2TestBase.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveV2DelegatesGasRebate_20230703} from 'src/AaveV2DelegatesGasRebate_20230703/AaveV2DelegatesGasRebate_20230703.sol';
+import {IERC20} from 'lib/solidity-utils/src/contracts/oz-common/interfaces/IERC20.sol';
+
+contract AaveV3EthWETHPayloadTest is ProtocolV2TestBase {
```

Comment:
not important, but i would name it after payload

Reply:
done ser